packages/FormElement/src/components/Content/Content.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ContentContainerStyled from "./Content.styles";
+
+const propTypes = {
+  children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
+  idForLabel: PropTypes.string,
+  refLabel: PropTypes.shape({ current: PropTypes.instanceOf(Object) }),
```

Comment:
Would be sweet to have this as one of our customPropTypes... 😄

Reply:
Ok I've added